recipes/gpm_imerg/recipe.py

```
@@ -109,18 +108,11 @@ def earthdata_auth(username: str, password: str):
 fsspec_open_kwargs = earthdata_auth(ED_USERNAME, ED_PASSWORD)
 
 
-# Remove method when https://github.com/pangeo-forge/pangeo-forge-recipes/pull/556/files is merged.
-@beam.ptransform_fn
-def ConsolidateMetadata(pcoll: beam.PCollection) -> beam.PCollection:
-    """Consolidate metadata into a single .zmetadata field.
-    See zarr.consolidate_metadata() for details.
-    """
-
-    def _consolidate(store: zarr.storage.FSStore) -> zarr.storage.FSStore:
-        zarr.consolidate_metadata(store, path=None)
-        return store
-
-    return pcoll | beam.Map(_consolidate)
+def test_ds(store: zarr.storage.FSStore) -> None:
+    import xarray as xr
+    ds = xr.open_dataset(store, engine="zarr", chunks={})
+    for dim, size in ds.dims.items():
+        print(f"Dimension: {dim}, Length: {size}")
```

Do we want to throw an exception?